README.md

```diff
@@ -1,117 +1,121 @@
- - Update `VERSION` in `python/setup.py` with the same version that you will use for the tag (manual step required until [#2](https://github.com/src-d/lookout-sdk/issues/2) is implemented).
- - Create the release tag.
-
- # License
-[Apache License v2.0](./LICENSE)
+<p align="center">
+  <a href="https://github.com/src-d/lookout-sdk/releases">
+    <img src="https://badge.fury.io/gh/src-d%2Flookout-sdk.svg"
+         alt="GitHub version">
+  </a>
+  <a href="https://pypi.org/project/lookout-sdk/">
+    <img src="https://badge.fury.io/py/lookout-sdk.svg"
+         alt="PyPI version">
+  </a>
+  <a href="https://travis-ci.org/src-d/lookout-sdk">
+    <img src="https://travis-ci.org/src-d/lookout-sdk.svg?branch=master"
+         alt="Build Status">
+  </a>
+  <a href="https://godoc.org/gopkg.in/src-d/lookout-sdk.v0/pb">
+    <img src="https://godoc.org/gopkg.in/src-d/lookout-sdk.v0?status.svg"
+         alt="GoDoc">
+  </a>
+  <a href="https://docs.google.com/document/d/1pqz-_SHO5BsJE-aa8o_bAY3r5vR67amnWN8-qZc2UgY">
+    <img src="https://img.shields.io/badge/source%7Bd%7D-design%20document-blue.svg"
+         alt="source{d} design document">
+  </a>
+</p>
+
+<p align="center"><b>
+    <a href="https://www.sourced.tech">Website</a> •
+    <a href="https://docs.sourced.tech">Documentation</a> •
+    <a href="https://blog.sourced.tech">Blog</a> •
+    <a href="http://bit.ly/src-d-community">Slack</a> •
+    <a href="https://twitter.com/sourcedtech">Twitter</a>
+</b></p>
+
+**Table of Contents**
+
+<!--ts-->
+   * [What does the SDK provide?](#what-does-the-sdk-provide)
+   * [Caveats](#caveats)
+      * [DataService](#dataservice)
+   * [Contributing](#contributing)
+      * [Community](#community)
+   * [Code of Conduct](#code-of-conduct)
+   * [License](#license)
+
+<!-- Added by: david, at: 2018-12-07T15:43+01:00 -->
+
+<!--te-->
+
+
+# What does the SDK provide?
+
+_For the complete documentation of **source{d} Lookout**, please take a look into [http://docs.sourced.tech/lookout](http://docs.sourced.tech/lookout)._
+
+_For detailed information about the different parts of Lookout, and how they interact you can go to the [lookout architecture guide](https://github.com/src-d/lookout/blob/master/docs/architecture.md)._
+
+**lookout-sdk** provides:
+
+- **proto [definitions](./proto)**,
+- pre-generated libraries code for [Golang](./pb) and [Python](./python):
+  - an easy **access to the DataService API though a gRPC** service to lookout analyzers, so they won't need to deal with actual Git repositories, UAST extraction, programming language detection, etc,
+  - low-level **helpers to workaround some protobuf/gRPC caveats**,
+- quickstart [examples](./examples) of an Analyzer that detects language and number of functions (written in Go and in Python).
+
+
+# Caveats
+
+When the gRPC client and servers are being used, it is needed:
+- to set [max gRPC message size](https://github.com/grpc/grpc/issues/7927); using lookout-sdk helpers it can be done:
```

Comment:
maybe we need to explain why we change defaults. (UAST)

Reply:
I agree.

Just to confirm that I understood it, I'd reword as:
>_to set a shared maximum gRPC message size in gRPC servers and clients (to avoid hitting different gRPC limits [see [grpc/grpc#7927](https://github.com/grpc/grpc/issues/7927)]); the `Analyzer` service should do so when dialing `DataService` and serving `NotifyReviewEvent` and `NotifyPushEvent`; using lookout-sdk helpers it can be done: